step.yml

```diff
@@ -1,14 +1,35 @@
 title: Run CocoaPods install
 summary: |-
-  You don't need to keep the installed pods in your repo, simply add this step
+  You don't need to keep the installed pods in your repo. Simply add this Step
   and Bitrise will do the pod install for you on the VM!
 description: |-
-  Run CocoaPods install (`$ pod install`) in your App's directory.
 
-  CocoaPods version is determined based on the Podfile.lock or
-  on the Gemfile.lock (if Podfile.lock does not exist in the Podfile's
-  directory and Gemfile.lock contains cocoapods gem)
-  in the Podfile's directory.
+  CocoaPods is a dependency manager for Swift and Objective-C Cocoa projects. This Step uses CocoaPods' `pod install` command to install your dependencies on the virtual machine where your Bitrise build runs. CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file if Podfile.lock does not exist in the Podfile's
+  directory and Gemfile.lock contains the `cocoapods` gem).
```

Comment:
If the Cocoapods version is specified in the Gemfile.lock, and it the version matches the Cocoapods version in thePodfile.lock (the version range specified contains it); then the Gemfile.lock Cocoapods version will be installed and used (using bundler).
Otherwise the Cocoapods version in the Podfile.lock will be installed as a global gem.
If no Cocoapods version is defined in the Podfile.lock, the preinstalled sytem Cocoapods version will be used.